src/studio/src/designer/Dockerfile

```diff
@@ -56,14 +56,14 @@ RUN npm ci
 COPY src/designer/backend .
 RUN npm run gulp build
 
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1.404-alpine3.12 AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1.405-alpine3.12 AS build
 COPY src/designer/backend ./designer/
 COPY --from=generate-designer-js /wwwroot ./designer/wwwroot
 
 RUN dotnet build designer/Designer.csproj -c Release -o /app_output
 RUN dotnet publish designer/Designer.csproj -c Release -o /app_output
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1.10-alpine3.12 AS final
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1.11-alpine3.12 AS final
```

Comment:
@SandGrainOne Is there a reason for not going for the latest .NET 5 in Designer?

Reply:
This is planned for next week